src/pyodide-py/pyodide/_base.py

```diff
@@ -49,55 +49,38 @@ class CodeRunner:
 
     Examples
     --------
-    >>> CodeRunner("1+1", {}).run()
+    >>> CodeRunner().run("1+1")
     2
-    >>> CodeRunner("1+1;", {}).run()
+    >>> CodeRunner().run("1+1;")
     """
 
-    def __init__(self, code: str, ns: Dict[str, Any]):
+    def __init__(self, ns: Dict[str, Any] = {}):
         """
         Constructor.
 
         Parameters
         ----------
-        code
-           the Python code to run.
         ns
            `locals()` or `globals()` context where to execute code.
         """
```

Comment:
This docstring should be merged with class docstring. `__init__` doesn't need one.

Also we should add a note that after the first call to `run`, the ns will be modified inplace (as far as I can tell) which will impact subsequent code evaluation.

Reply:
Right. Is `Parameters` allowed in class docstring?

Reply:
yes, https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes